sections/main-search.liquid

```diff
@@ -1,13 +1,11 @@
-{{ 'template-collection.css' | asset_url | stylesheet_tag }}
+{{ 'component-card.css' | asset_url | stylesheet_tag }}
+{{ 'component-price.css' | asset_url | stylesheet_tag }}
+{{ 'component-product-grid.css' | asset_url | stylesheet_tag }}
```

Comment:
I'm wondering something. All those files seem to also be using the `component-card.css` asset. Could it make sense to add the content of `component-product-grid.scss` into it ? Or because it's pretty specific we want to keep it on its own ? 
I was just thinking it could be one less request.

Reply:
I see what you mean. The card component is used on multiple place such as blog post, search template result, list collection section, collage section, but the product grid component isn't necessarily always together with the card component, so that's why I left it separately.

Reply:
Sound good